airflow/providers/cloudant/provider.yaml

```
@@ -25,7 +25,7 @@ state: ready
 source-date-epoch: 1723969866
 # note that those versions are maintained by release manager - do not update them manually
 versions:
-  - 3.6.0
+  - 4.0.0
   - 3.5.2
```

```suggestion
  - 4.0.0
  - 3.6.0
  - 3.5.2
```

@topherinternational you have wrong edit here. you removed 3.6.0, please restore it.